package.json

```diff
@@ -4,15 +4,23 @@
   "description": "nodejs tutorial project",
   "main": "index.js",
   "scripts": {
-    "start": "node ./main.js",
-    "test": "node ./test.js"
+    "start": "node ./dist/main.bundle.js",
+    "test": "node ./test.js",
+    "build": "npm run webpack",
```

Comment:
Do you realy need webpack for compilation? Maybe it's better to use ts cli? See [tsc](https://www.typescriptlang.org/docs/handbook/compiler-options.html)